report-generator/src/main/java/com/teamscale/report/testwise/model/TestInfo.java

```diff
@@ -38,6 +38,11 @@ public class TestInfo {
 	/** All paths that the test did cover. */
 	public final List<PathCoverage> paths = new ArrayList<>();
```

Comment:
What about `PathCoverage`? Please go through all the serialized class in a structured manner and ensure we didn't miss any additional ones.

Reply:
Done. Generally, I tried to cover all classes that are passed to `.adapter`.